projects/jsoncpp/build.sh

```diff
@@ -28,3 +28,17 @@ $CXX $CXXFLAGS -I../include $LIB_FUZZING_ENGINE \
 
 # Add dictionary.
 cp $SRC/jsoncpp/src/test_lib_json/fuzz.dict $OUT/jsoncpp_fuzzer.dict
+
+# Compile json proto.
```

Comment:
this part is also not needed in the MSan build (i.e. move these three lines below under the `if`)